2-structured-data/test/_api-tests.js

```diff
@@ -13,57 +13,52 @@
 
 'use strict';
 
-const getRequest = require(`@google-cloud/nodejs-repo-tools`).getRequest;
-const test = require(`ava`);
+const getRequest = require('@google-cloud/nodejs-repo-tools').getRequest;
```

Comment:
Are we sure that `@google-cloud/nodejs-repo-tools` is not an environment variable.. If it is an external variable than using single quote would not create problems.

Reply:
It is npm module.

Reply:
oh, My bad